server/main.cpp

```diff
@@ -88,8 +108,9 @@ int main(int argc, char *argv[]) {
             tcp::socket socket(ioc);
             acceptor.async_accept(socket, yield);
             BOOST_LOG_SEV(logger, sev::info) << "Got connection";
+            nibaserver::db_accessor db(connector);
```

Comment:
This db probably goes out of scope and gets destructed? Please double check

Or move it into the session(see how we do it for socket_ in server_session)